client/extensions/woocommerce/components/image-thumb/index.js

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const ImageThumb = ( { width, height, src, alt, ...props } ) => {
+	const style = {
+		width,
+		height,
+		minHeight: height,
+	};
+
+	const imageClasses = classNames( 'image-thumb', {
+		hasImage: src,
+	} );
+	const placeholderClasses = classNames( 'image-thumb__placeholder' );
+
+	if ( ! src ) {
+		return (
+			<div className={ imageClasses }>
+				{' '}
```

Comment:
Why do we need the spaces here?

Reply:
Uhh, I'm actually not sure how those ended up there but they are not needed. Thanks for catching.

Reply:
👍 i think the prettier formatting adds them when it wraps lines of text, maybe that's what happened here.